mypy-baseline.txt

```diff
@@ -1,6 +1,38 @@
 ee/api/test/base.py:0: error: "setUpTestData" undefined in superclass  [misc]
 ee/api/test/base.py:0: error: Incompatible types in assignment (expression has type "None", variable has type "License")  [assignment]
+ee/api/test/test_billing.py:0: error: "_MonkeyPatchedResponse" has no attribute "url"  [attr-defined]
```

Comment:
:(

Reply:
Yeah, weirdly none of the ee test files were in mypy 🤯